rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerRestClient.java

```diff
@@ -429,20 +434,29 @@ public ListFile list(String sessionId, String dataspacePath, String pathname) th
         }
     }
 
-    private JobIdData submit(String sessionId, InputStream job, MediaType mediaType, Map<String, String> variables)
-            throws Exception {
+    private JobIdData submit(String sessionId, InputStream job, MediaType mediaType, Map<String, String> variables,
+            Map<String, String> genericInfos) throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)

Reply:
Ok